Nos/Views/Home/HomeFeedView.swift

```diff
@@ -63,25 +67,33 @@ struct HomeFeedView: View {
 
     var body: some View {
         ZStack {
-            PagedNoteListView(
-                refreshController: $refreshController,
-                databaseFilter: homeFeedFetchRequest,
-                relayFilter: homeFeedFilter,
-                relay: selectedRelay,
-                managedObjectContext: viewContext,
-                tab: .home,
-                header: {
-                    EmptyView()
-                },
-                emptyPlaceholder: {
-                    VStack {
-                        Text(.localizable.noEvents)
-                            .padding()
+            VStack(spacing: 8) {
+                TipView(welcomeTip)
+                    .padding(.top, 20)
+                    .padding(.horizontal, 16)
+                    .tipBackground(LinearGradient.horizontalAccentReversed)
+                    .tipViewStyle(.inline)
+
+                PagedNoteListView(
+                    refreshController: $refreshController,
+                    databaseFilter: homeFeedFetchRequest,
+                    relayFilter: homeFeedFilter,
+                    relay: selectedRelay,
+                    managedObjectContext: viewContext,
+                    tab: .home,
+                    header: {
+                        EmptyView()
```

Comment:
I wish we could put the tip view in here, so that it's inside the scroll view if the user decides to scroll right away. I tried it and it works but it leaves an awkward blank space at the top after you dismiss the tip. I'm not sure if this is due to the padding on the tip view or what. I don't see an easy way to fix it but maybe you do. I don't feel like it's a blocker though.

Reply:
Yeah, I tried but couldn't find a good solution. I'll leave it for now and we can circle back later.